src/ontology/mondo-edit.obo

```diff
@@ -526173,6 +526211,18 @@ is_a: MONDO:0019042 {source="https://clinicalgenome.org/affiliation/40130/"} ! m
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7976 {source="https://clinicalgenome.org/affiliation/40130/"} ! NR2F2
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6666" xsd:anyURI
 
+[Term]
+id: MONDO:0800459
+name: noxacusis or pain hyperacusis
```

Comment:
Should we be creating term names that include "or" in the name especially when each side of the "or" statement is also included as synonyms?

Also, is this a disease vs. a phenotype?

Reply:
revise label